TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/entities/Excursion.java

```diff
@@ -167,26 +166,23 @@ public Trip getTrip() {
      */
     public void setTrip(Trip trip) {
         this.trip = trip;
+        trip.addExcursion(this);
```

Comment:
@filipceko i put this for sync to trip as Simone comment before,is it okey?

Reply:
Jop, seen that, just next time plese inform me before you do this because I could not figure out why couldn't I push :D